electricpy/__init__.py

```diff
@@ -4782,6 +4782,102 @@ def wireresistance(length=None,diameter=None,rho=16.8*10**-9,R=None):
     elif R != None and length != None:
         A = rho*length/R
         return _np.sqrt(4*A/pi)
+
+def parallel_plate_capacitance(A=None, d=None, e=e0, C=None):
+    r"""
+    Parallel-Plate Capacitance Calculator.
+
+    Enter three values to calculate the remaing one. Even though every variable
+    is unitless, please use the International System of Units.
+
+    .. math:: C =\frac{\varepsilon A}{d}
+
+    Parameters
+    ----------
+    A:  [float]
+        Area of the plate, unitless.
+    d:  [float]
+        Distance between the plates, unitless.
+    e:  [float]
+        Permitivity of the dielectric, unitless.
+        Default value is the permittivity of free space: 8.854E-12
+    C:  [float]
+        Capacitance, unitless.
+    Returns
+    -------
+    A:  [float]
+        Area of the plate, unitless.
+    d:  [float]
+        Distance between the plates, unitless.
+    e:  [float]
+        Permitivity of the dielectric, unitless.
+        Default value is the permittivity of free space: 8.854E-12
+    C:  [float]
+        Capacitance, unitless.
+    """
+    if C == A == d == None:
+        raise ValueError("To few arguments.")
+    # Given area and distance
+    if A != None and d != None:
+        return e*A/d
+    # Given capacitance and distance
+    elif C != None and d != None:
+        return d*C/e
+    # Given capacitance and area
+    elif C != None and A != None:
+        return e*A/C
+
+def solenoid_inductance(A=None, l=None, N=None, u=u0, L=None):
+    r"""
+    Solenoid Inductance Calculator.
+
+    Enter four values to calculate the remaing one. Even though every variable
+    is unitless, please use the International System of Units.
+
+    .. math:: L = \frac{\mu N^2 A}{l}
```

Comment:
consider using `\cdot` to denote multiplication in the numerator, it makes things explicitly clear that there's multiplication taking place